frontend/src/components/data_entry/subsections/CheckboxesRadioSubsection.module.css

```
@@ -2,9 +2,14 @@
   width: 37.5rem;
   margin-bottom: 2rem;
 
-  legend {
+  legend,
+  p {
     font-size: var(--font-size-md);
     color: var(--gray-900);
     margin-bottom: 1rem;
   }
+
+  legend {
+    font-weight: 700;
```

```suggestion
    font-weight: bold;
```
400 and 700 preferably as words, so it's easier to read and more in line with the existing styling